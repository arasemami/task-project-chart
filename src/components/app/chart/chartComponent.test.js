import React from 'react';
import ChartComponent from './chartComponent';
import renderer , { create }  from 'react-test-renderer'
 

test('rendersChartComponent', () => {
    const tree = renderer.create(<ChartComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });