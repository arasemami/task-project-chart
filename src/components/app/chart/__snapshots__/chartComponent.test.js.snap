// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendersChartComponent 1`] = `
<div
  className=""
>
  <div
    className="mixed-chart"
  >
    <div />
  </div>
</div>
`;
